### PR TITLE
Turning on blunderbuss on the docs submit-queue.

### PR DESCRIPTION
**Pull request by foxish**

### CODE REVIEW

File: mungegithub/submit-queue/deployment/docs/configmap.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ data:
   config.http-cache-dir: /cache/httpcache
   config.organization: kubernetes
   config.project: kubernetes.github.io
-  config.pr-mungers: size
```
Comment (apelisse) [inline mungegithub/submit-queue/deployment/docs/configmap.yaml]: Do we not care about size?

Comment (foxish) [inline mungegithub/submit-queue/deployment/docs/configmap.yaml]: We cannot turn it on before the check-labels munger creates the size labels for this repository. We'll turn it back on after that is done.

**Comment by foxish** [comment]
@lavalamp What would be the right thing to do here? If we have no e2e tests, should we disable the `@k8s-bot test this` comment and merge as soon as we get an LGTM?

**Comment by apelisse** [comment]
Wait, it must at least wait for travis-ci to pass.

**Comment by foxish** [comment]
travis-ci is a [required context]((https://github.com/kubernetes/contrib/blob/master/mungegithub/submit-queue/deployment/contrib/configmap.yaml#L19) in the contrib deployment, so we do wait for it to pass before we merge.

**Comment by apelisse** [comment]
I got scared by the "merge as soon as we get an LGTM" :-)

Cool!

**Pull request merged**